newswires/client/src/WireItemTable.tsx

```diff
@@ -0,0 +1,125 @@
+import {
```

Comment:
this is a big diff in the Github UI, due to the file being renamed, but I did the renaming in a separate commit so looking at the first commit will hopefully give a nicer diff of the main substantive changes